tests/ballerina-test-integration/src/test/resources/httpService/httpServiceConfigTest.bal

```diff
@@ -0,0 +1,34 @@
+import ballerina.net.http;
```

Comment:
Instead of having a integration test can we make it unit test ? So that we can avoid the transport. Transport already has a testcase for this.

Reply:
Since this is a service configuration shouldn't we have an integration test? I think integration tests need to be written for other service configurations as well. Correct me if I am mistaken.

Reply:
Well, we don't have to. Because transport bit is tested at transport and the ballerina bit should be tested at ballerina. When testing the ballerina bit we shouldn't have to test the transport bit as well. Then there is a problem of separation of concerns :) .

What we are trying to archive is less number of integration tests and more number of unit tests similar to test-pyramid.